### ISSUE TITLE
sugestion, provide a shortcut key and automatically advertise hortographic errors in google docs

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8155]
### Steps to reproduce:
Open a document in google docs.
When navigating with arrows the nvda does not announce the hortographic errors.
There is not even a nvda shortcut key to navigate between the horizonal errors in the document, as there is in word (w).

### Expected behavior:
The suggestion is that the nvda read the hortographic errors when navigating with the arrows through the text.
I would also like to suggest a shortcut key to allow you to navigate the horizons in a text.

### Actual behavior:
Nvda does not currently advertise the hortographic errors when navigating the text with the arrows.
There is no shortcut key to allow quick movement between hortographic errors in text.

### System configuration:
NVDA version:
all

NVDA Installed or portable:
instaled and portable

Other information:
> Example: Running in a VM

Windows version:
all

Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?
yes

Have you tried any other versions of NVDA?
> Please list them and the result

**Comment by ehollig** [comment]
@fernando-jose-silva what exactly do you mean when you say, "hortographic errors"? According to the [shortcuts for Google Docs](https://support.google.com/docs/answer/179738?co=GENIE.Platform%3DDesktop), you should be able to navigate to the next misspelling or previous misspelling by pressing Ctrl + ' or Ctrl + ;. I do not believe NVDA can create shortcuts for Google Docs.

**Comment by fernando-jose-silva** [comment]
Many thanks for the feedback.
Is it possible to implement in nvda the ability to report the horological and grammatical errors reported by google docs when browsing line by line or word by word, functionality that we have in word.

**Comment by leonardder** [comment]
What do you mean with horological errors?

NVDA should already be able to report grammar errors when appropriately exposed, see #8280

**Comment by leonardder** [comment]
I think I understand you now and I've been able to reproduce this. Google uses its own implementation to announce spelling errors, it does not expose spelling and grammar errors in a way that NVDA can detect them as such. Therefore, there's probably nothing we can do here.

@derekriemer: Could you spread any light on this?

**Comment by frastlin** [comment]
I request this feature every day I use Google docs.
This, jumping by headings, and editing tables, are still not the best experience for me in Google Docs.

**Comment by Adriani90** [comment]
maybe @aleventhal or @ObjectInSpace can help here, NVDA should be able to identify these if they are properly exposed I guess.

**Comment by ObjectInSpace** [comment]
I'm happy to pass this on, but I'm a little confused about the precise ask. I was under the impression that spelling and grammatical errors are currently announced by NVDA if docs is in braille mode. Is this not the case?

Note that with braille mode off, docs uses live regions to convey content updates. This is unlikely to change. However, browsers read spelling and grammar error markers for text in input fields and content editables, such as gmail's body field. So if braille mode is turned on from docs > tools menu > accessibility settings, this should work there as well.

Is there a third type of error that is not getting reported? A reproduction case might make things more clear,

Thanks much!

**Comment by Adriani90** [comment]
Ok this works properly in Firefox but not completely in Chrome. When enabling braille support, in Chrome, NVDA reports spelling and gramar errors when navigating through the document, but it does not report them when you type via sound or something like it does in Firefox.
